crates/iota-node/src/lib.rs

```diff
@@ -2047,6 +2045,12 @@ impl IotaNode {
         if let Some(validator_components) = &*self.validator_components.lock().await {
             validator_components.consensus_manager.shutdown().await;
         }
+
+        // Shutdown the gRPC server if it's running
+        if let Some(grpc_handle) = &self.grpc_server_handle {
+            info!("Shutting down gRPC server");
+            grpc_handle.server_handle.abort();
```

Comment:
We have `shutdown` method.
```suggestion
            grpc_handle.shutdown();
```

Reply:
Oh looks like we have this error so I explicitly abort it
```
cannot move out of `*grpc_handle` which is behind a shared reference
move occurs because `*grpc_handle` has type `GrpcServerHandle`, which does not implement the `Copy` traitrustc[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic%20message%20[9]?9#file:///Users/bylin/Projects/IOTA/iota/crates/iota-node/src/lib.rs)
lib.rs(2052, 25): `*grpc_handle` moved due to this method call
server.rs(32, 21): `GrpcServerHandle::shutdown` takes ownership of the receiver `self`, which moves `*grpc_handle`
```

Reply:
Added the gracefully shutdown at [2b788eb](https://github.com/iotaledger/iota/pull/8019/commits/2b788ebb9db76429cbbbce6255ffac9c32f02cf7).